courses/machine_learning/deepdive2/structured/solutions/4a_sample_babyweight.ipynb

```diff
@@ -0,0 +1,2808 @@
+{
```

Comment:
<p>A little space can be save here using kwargs like so</p><p> </p><p>csv_kwargs = {"columns": columns, "header": False, "index": False}</p><p>train_df.to_csv(path_or_buf="train.csv", **csv_kwargs)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/671/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>I wanted it to be clear to all students, some of which aren't as familiar with all python things such as unpacking operators. I'd rather have an extra line or two.</p>

Reply:
That's a valid point, but it's the same level of python fluency as list comprehension. I think it would be good to be consistent with [earlier labs](https://github.com/GoogleCloudPlatform/training-data-analyst/blob/master/courses/machine_learning/deepdive/02_tensorflow/c_estimator.ipynb) and teach newbies a cool new trick.

Reply:
I'd say it is a little more obscure than list comprehensions. I don't see any unpacking operators in the lab you linked. Anyway, this seems more like a nit and personal preference so I think this looks good as is.

Reply:
It's ok if you want to leave it as is for now, Ryan, but to warn your when you go for Google readability, they have a whole sections dedicated to practices like this called "Python Idioms".

Reply:
<p>Ya, I'm leaving it as is. I know the Python idioms, but this is not the right place to be using all of those to possibly confuse the students. Students aren't going to for Google readability ;)</p>